src/style.css

```
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+}
+
+:root {
+  --moyo-blue: #00acdc;
+}
+
+.site-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo-link {
+  align-self: center;
+  height: 40px;
+}
+
+.logo-link img {
+  height: 40px;
+  display: block;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav__item {
+  height: 100%;
+  margin: 0 10px;
+  list-style: none;
+  align-content: center;
+  font-size: 12px;
+  position: relative;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: inline-block;
+  line-height: 60px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
+}
+
+.nav__link:hover {
+  color: var(--moyo-blue);
+}
+
+.nav__link.is-active {
```

since `is-active` class is used exclusively on `nav__link` you can simplify selector to just `.is-active`
```suggestion
.is-active {
```